### PR TITLE
[Feature] Add click support for onboarder dropzone in IE

### PR DESCRIPTION
**Pull request by mfraezz**
Purpose
=======
Partial fix for https://github.com/CenterForOpenScience/osf.io/issues/1669
Adds click-to-upload functionality, does not add drag&drop functionality. There's something weird about IE's event system, preventing drag&drop related events from being registered. It may be due to an interaction with knockout, but I've found no evidence for or against that.

Changes
=======
* One `dropzoneOpts` key-value pair added
* Change the header above the dropzone (to `"Click below to select file"`) iff IE

Side Effects
=========
May fix similar issue for Safari 7.0.3 (untested). See https://github.com/CenterForOpenScience/osf.io/issues/1250
If so, header will not change in Safari.

### CODE REVIEW

**Comment by lyndsysimon** [comment]
I spoke with Matt while he was working on this. It's ugly, but neither of us could come up with a better way.

Passing it to you, @sloria. Your call.

File: website/static/js/onboarder.js [hunk 1]
```diff
@@ -596,6 +597,10 @@ function OBUploaderViewModel(params) {
     };
     self.dropzone = new Dropzone(self.selector, dropzoneOpts);
 
+    if($osf.isIE()){
+        $('h4:contains(1. Drop file (or click below))').replaceWith('<h4>1. Click below to select file</h4>');
```
Comment (sloria) [inline website/static/js/onboarder.js]: Rather than relying on the text of the h4, why not use an ID?

File: website/static/js/onboarder.js [hunk 2]
```diff
@@ -596,6 +597,10 @@ function OBUploaderViewModel(params) {
     };
     self.dropzone = new Dropzone(self.selector, dropzoneOpts);
 
+    if($osf.isIE()){
```
Comment (sloria) [inline website/static/js/onboarder.js]: Any workarounds like this should be accompanied by a code comment explaining why it's necessary.

**Comment by sloria** [comment]
Pass finished. :video_camera:

File: website/static/js/onboarder.js (hunk 1 above)
Comment (mfraezz) [inline website/static/js/onboarder.js]: I considered that first, but @lyndsysimon suggested this method. I'll use an ID if you think that's best.

**Pull request merged**